### PR TITLE
sbcl: update to 2.2.10

### PR DESCRIPTION
**Pull request by kakuhen**
#### Description
Minor version bump

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.7 19H2026 x86_64
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by kakuhen** [comment]
@catap Will this conflict with your changes? If so, then I can update my commit message so that it comes after your changes.

**Comment by catap** [comment]
It will conflicts, yeah

**Comment by kakuhen** [comment]
Will it be safe to merge your PR first then mine? My diffs only touch the versions and checksums. But otherwise I want to believe our diffs are "isolated" and can be merged together.

**Comment by catap** [comment]
@kakuhen or you may assume commit from my PR into your. After that I'm fine with closing my PR  :)

**Comment by kakuhen** [comment]
Ok. I have resolved the conflicts. Feel free to merge when ready

File: math/maxima/Portfile [hunk 1]
```diff
@@ -39,7 +39,7 @@ subport maxima-devel {
     # Date:  Sat Oct 29 16:24:21 2022 +0000
     # commit 5b42cccfd9eee54d0458d8f4e63e6b44c55d48a1
     version     5.46-dev-20221029
-    revision    0
+    revision    1
```
Comment (catap) [inline math/maxima/Portfile:42]: are you sure that the only port which should be revbump is maxima?

Comment (kakuhen) [reply math/maxima/Portfile:42]: To be frank, I don't understand why Maxima needs to be bumped at all. It's extremely rare for Common Lisp software to depend on a specific version of SBCL, unless they are just loading FASLs directly into SBCL. In that case, yes we should revbump the port. In any case, I think Maxima is the only software that potentially needs revbump.

**Pull request merged**